.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,60 @@
+name: Publish cpu dockers
+on:
+  workflow_dispatch
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: TissueImageAnalytics/tiatoolbox
+  image: ghcr.io/tissueimageanalytics/tiatoolbox
+  TOOLBOX_VER: 1.3.0
```

Comment:
What will happen when we update the version?

Reply:
@shaneahmed This should 🤞 be updated by bump2version. But we may need to make sure this file is in the paths which is checks for version strings.

Reply:
Please check my comment here: https://github.com/TissueImageAnalytics/tiatoolbox/pull/504#discussion_r1018959664


I'm not sure how necessary is this and I don't know how this can be realized.